### ISSUE TITLE
pulmonary arterial medial hypertrophy

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3301]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

pulmonary arterial medial hypertrophy

>2. Synonyms (e.g., absent spleen)

Hypertrophy of the pulmonary artery wall

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

Increase in mass of the tunica media of the arteries in the pulmonary circulation. (PMID:25593290)

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal pulmonary artery morphology (MP:0000484)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174
--

**Comment by anna-anagnostop** [comment]
Added new term:
pulmonary arterial medial hypertrophy MP:0031193
